docker-compose.yml

```diff
@@ -21,6 +21,43 @@ services:
     ports:
       - 6379:6379
 
+  flask:
+    build:
+      context: .
+      dockerfile: Dockerfile-python
+    image: distributed-replays-python
+    # The sleep is less than ideal but there seems to be a delay between
+    # when dockerize can connect to postgres and when our backend can
+    command: sh -c "dockerize -timeout 30s -wait tcp://ccgg-postgres10:5432 -wait tcp://ccgg-redis4:6379 && sleep 3 && python ./RLBotServer.py"
```

Comment:
You can use [wait-for-it](https://github.com/vishnubob/wait-for-it) to handle waiting till the network endpoints come up

Reply:
I'm using alpine images for everything because they're much smaller then their Debian equivalents (the Python one is less than 1/10th the size). Unforunately wait-for-it doesn't work in alpine containers (I've spent multiple hours trying to make it work in the past), so I went with dockerize.